### PR TITLE
update entity management package

### PR DESCRIPTION
**Pull request by genric**

### CODE REVIEW

File: deploy/packages/bbp-packages.yaml [hunk 1]
```diff
@@ -46,7 +46,7 @@ packages:
     specs:
       - py-bbp-analysis-framework@0.6.39
       - py-simwriter@0.4.3
-      - py-entity-management@1.0.11
+      - py-entity-management@1.0.12
```
Comment (matz-e) [inline deploy/packages/bbp-packages.yaml:49]: why is this needed for Python 2 still?

Comment (genric) [reply deploy/packages/bbp-packages.yaml:49]: Because it is used with the simwriter.

Comment (matz-e) [reply deploy/packages/bbp-packages.yaml:49]: Fine.

**Review by matz-e — APPROVED** [review]
Sim-writer should move to Python 3.
